model_compression_toolkit/core/runner.py

```diff
@@ -122,7 +124,7 @@ def core_runner(in_model: Any,
             bit_widths_config = core_config.mixed_precision_config.configuration_overwrite
 
     else:
-        bit_widths_config = None
+        bit_widths_config = []
```

Comment:
Doesn't set_bit_widths (in line 129) expect None if a single precision is used?

Reply:
I don't think so. It uses a flag "mixed_precision_enable" to decide whether to use the given bit-width config or not